03_launch_mgmt_cluster.sh

```diff
@@ -28,20 +28,23 @@ export GOPATH
 M3PATH="${GOPATH}/src/github.com/metal3-io"
 BMOPATH="${M3PATH}/baremetal-operator"
 RUN_LOCAL_IRONIC_SCRIPT="${BMOPATH}/tools/run_local_ironic.sh"
-CAPM3PATH="${M3PATH}/cluster-api-provider-baremetal"
-
-BMOREPO="${BMOREPO:-https://github.com/metal3-io/baremetal-operator.git}"
-BMOBRANCH="${BMOBRANCH:-master}"
-CAPM3REPO="${CAPM3REPO:-https://github.com/metal3-io/cluster-api-provider-baremetal.git}"
 
 if [ "${CAPI_VERSION}" == "v1alpha3" ]; then
+  CAPM3PATH="${M3PATH}/cluster-api-provider-metal3"
   CAPM3BRANCH="${CAPM3BRANCH:-master}"
+  CAPM3REPO="${CAPM3REPO:-https://github.com/metal3-io/cluster-api-provider-metal3.git}"
```

Comment:
May be default it to something and once it is not default change it rather than default the env variable for each different case?

Reply:
Agree, updated